repo/packages/E/elasticsearch/0/config.json

```diff
@@ -0,0 +1,41 @@
+{
+  "properties": {
+    "mesos": {
+      "depscription": "Mesos specific configuration properties",
+      "properties": {
+        "master": {
+          "default": "zk://master.mesos:2181/mesos",
+          "description": "The URL of the Mesos master. This should point to the Mesos ZNode on the Zookeeper machine.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "master"
+      ],
+      "type": "object"
+    },
+    "elasticsearch": {
+      "depscription": "Elasticsearch specific configuration properties",
+      "properties": {
+        "instances": {
+          "default": "3",
+          "description": "The number of elasticsearch nodes to be instantiated. It is recommended that you provide an odd number, to prevent split brain issues.",
+          "type": "string"
+        },
+        "port": {
+          "default": "8080",
+          "description": "The port number of the ES-Mesos management front-end. Defaults to 8080.",
+          "type": "string"
+        },
+        "ram": {
+          "default": "2048",
+          "description": "The amount of ram that Mesos shall give give to the executors.",
+          "type": "string"
+        }
+      },
+      "required": [
+      ],
```

Comment:
required sometimes confuses developers.  It is all the properties that do NOT have guard conditions that are used in the marathon.json which ends up being most properties.

example of guard condition:  https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/C/cassandra/0/marathon.json#L53-L55